evenodd.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+
+int main()
+{
+	int y = 0;
+	int x;
+	printf("Please enter a number: \n");
+	scanf("%d", &x);
+
+	if (x%2 == y) {
+		printf("Your number is even.\n");
+	}
+	else {
+		printf("Your number is odd.\n");
+	}
```

Comment:
you forgot the return 0 and didn't include comments. But other than that, nice job.